### DISCUSSION TITLE
[Meta] Best frequency to run at?

### DISCUSSION THREAD

**Discussion by Nyarumi** [alexjustesen/speedtest-tracker#1778] [Q&A]
I Googled around and checked the issues here but I couldn’t find a real answer. 

I currently run the test every 4 hours, but I’d rather have hourly data. I have 2.5 gigabit internet which can use a lot of data when speedtesting, so I’ve been afraid to increase the frequency to hourly. I don’t have a data cap and I don’t think I’ve ever run into any throttling issues, but I’m still a little worried… What frequency does everyone use for their tests?

**Comment by svenvg93** [comment]
If you don't have a data cap you can test as much as you want 😅. 

Just keep in mind that testing might also influence your own use of the bandwidth, as in testing to much might cause buffering in live streaming or latency increase while online gaming. As well the other way around, your own bandwidth usage will influence the test, when downloading a big file or game update might influence the test results. 

Personally I test every second hour (6 */2 * * *), it gives me a nice balance of having enough data and not test to much. As the end of the day we pay to use the bandwidth, not just for testing it 😉 .

**Comment by alexjustesen** [comment]
Going to convert this to a discussion, not really an issue per say
